### PR TITLE
POC: esl select

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

File: src/modules/draft/esl-select/core/esl-select-dropdown.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  @bind
+  public updatePosition() {
+    if (!this.activator) return;
+    const windowY = window.scrollY || window.pageYOffset;
+    const rect = this.activator.getBoundingClientRect();
+
+    this.style.top = `${windowY + rect.y + rect.height}px`;
+    this.style.left = `${rect.left}px`;
+    this.style.width = `${rect.width}px`;
```
Comment (dshovchko) [inline src/modules/draft/esl-select/core/esl-select-dropdown.ts:78]: some method is required for formatted output of values

File: src/modules/draft/esl-select/core/esl-select-list.less [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    top: 56px;
+    bottom: 6px;
+  }
+
+  .esl-select-list-container {
+    flex: 1 1 auto;
+    height: 100%;
+    margin-right: 16px;
+
+    //border-right: 1px solid @esl-select-secondary;
```
Comment (dshovchko) [inline src/modules/draft/esl-select/core/esl-select-list.less:18]: comment

**Review by dshovchko — APPROVED** [review]

File: src/modules/draft/esl-select/core/esl-select-dropdown.ts (hunk 1 above)
Comment (ala-n) [reply src/modules/draft/esl-select/core/esl-select-dropdown.ts:78]: Yep, we will think about utils for CSS property rendering in the future.

**Comment by ala-n** [comment]
Dear ESL contributors, feel free to add comments or concerns if you find such for the current PR. Note that the conversation is still unlocked for contributors. Thank you!

**Pull request merged**